tests/python_client/testcases/test_collection.py

```diff
@@ -2429,7 +2429,7 @@ def test_load_replica_partitions(self):
         error = {ct.err_code: 1, ct.err_msg: f"not loaded into memory"}
         collection_w.query(expr=f"{ct.default_int64_field_name} in [0]",
                            partition_names=[ct.default_partition_name, ct.default_tag],
-                           check_task=CheckTasks.err_res, check_items=error)
+                           check_task=CheckTasks.check_query_empty)
```

Comment:
Why this needed to be modified to query successfully with empty results?

Load one partition, and I think it is right to return error if querying the non-loaded partition.